### ISSUE TITLE
Male sure examples actually work

### ISSUE THREAD

**Issue by junghans** [espressopp/espressopp#369]
Some of them haven't been touched for years.

We might at least want to byte compile them.

**Comment by XzzX** [comment]
What about running them on a weekly basis?

**Comment by junghans** [comment]
Sounds good.

They aren't integrated in CMake though.

**Comment by XzzX** [comment]
True, we could add them as tests with a specific label, if you want to run them via cmake. I will have a look at it.
